src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -180,6 +180,9 @@
 
     public static final String LEGACY_PERMIT_NAMESPACE = "PERMIT";
 
+    public static final String DOCUMENT_REINDEX_FILE_NAME_PREFIX = "documentReindex";
+    public static final String DOCUMENT_REINDEX_FILE_NAME_EXTENSION = ".txt";
```

Comment:
Should this constant have a more generic name such as "TEXT_EXTENSION"?
Asking if this would make sense as I can see where having the constant name defined as you currently have it is explicit for it's use but having it more generically defined would allow us to reuse/replace two other constants when that code is refactored.

Reply:
Earlier today when I went to CUKFSConstants to remove an unneeded constant in the other open PR, I noticed that we had actually started defining some constants for file extensions. I will rework this ".txt" constant to have a more generic name.

I also noticed that CUKFSConstants has two different strategies for defining those file extension constants. I will create a follow-up user story to get those consolidated into one spot.